src/commands/games.py

```diff
@@ -14,9 +16,10 @@ async def poker_message_check(reaction: Reaction, user: User):
     if room is None:
         return False
 
-    url = "http://84.107.225.27:5000/%d/game?id=%d" % (room.id, user.id)
+    url = "http://localhost:5000/%d/game?id=%d" % (room['message_id'], user.id)
```

Comment:
You dont have to change anything here but maybe we want to add a host and port to the config file